core/src/main/java/com/sequenceiq/cloudbreak/job/AbstractStackJobInitializer.java

```diff
@@ -1,34 +1,19 @@
 package com.sequenceiq.cloudbreak.job;
 
-import java.util.stream.Stream;
+import java.util.List;
 
 import javax.inject.Inject;
 
-import com.sequenceiq.cloudbreak.domain.projection.StackTtlView;
-import com.sequenceiq.cloudbreak.domain.stack.Stack;
 import com.sequenceiq.cloudbreak.quartz.model.JobInitializer;
+import com.sequenceiq.cloudbreak.quartz.model.JobResource;
 import com.sequenceiq.cloudbreak.service.stack.StackService;
 
 public abstract class AbstractStackJobInitializer implements JobInitializer {
 
     @Inject
     private StackService stackService;
 
-    protected Stream<Stack> getAliveStacksStream() {
-        return stackService.getAllAlive().stream()
-                .map(this::convertToStack);
-    }
-
-    protected Stream<Stack> getAliveAndNotDeleteInProgressStacksStream() {
-        return getAliveStacksStream()
-                .filter(s -> !s.isStackInDeletionOrFailedPhase());
-    }
```

Comment:
As discussed these should not be deleted completely as there are initializers (e.g. StructuredSynchronizerJobInitializer) that should init jobs for every cluster besides delete_completed status ones and not just alive ones.